lib/server/parse-websocket.js

```diff
@@ -32,5 +33,15 @@ function parseWebSocketRequest (socket, params) {
   )
   params.compact = -1 // return full peer objects (used for websocket responses)
 
+  params.ip = opts.trustProxy
+      ? socket.upgradeReq.headers['x-forwarded-for'] || socket.upgradeReq.connection.remoteAddress
+      : (socket.upgradeReq.connection.remoteAddress && socket.upgradeReq.connection.remoteAddress.replace(common.REMOVE_IPV4_MAPPED_IPV6_RE, '')) // force ipv4
+  params.port = socket.upgradeReq.connection.remotePort
+  if (params.port) {
+    params.addr = (common.IPV6_RE.test(params.ip) ? '[' + params.ip + ']' : params.ip) + ':' + params.port
```

Comment:
This `addr` property will cause issues in lib/server/swarm.js here: https://github.com/feross/bittorrent-tracker/blob/7406750b747de119ce9025855f945cb6ff998344/lib/server/swarm.js#L16

The presence of `addr` will cause that to be used instead of `peer_id` for webrtc peers.